### ISSUE TITLE
[BUG] [T6] Issue detecting missing ";"

### ISSUE THREAD

**Issue by arnauruana** [xensik/gsc-tool#97]
Using the [latest release](https://github.com/xensik/gsc-tool/releases/tag/1.0.583) there is a **bug** in the compiler functionality when **compiling T6 scripts** where some **missing ';'**  are **not detected** by it. Therefore it allows you to **compile syntactically incorrect scripts** causing the game to throw and **execution error** later.

Example working expectedly:
```cpp
// file path: "src/t6/test.gsc"

init()
{
    println("foo1");
    println("foo2")
}
```
```bash
./bin/gsc-tool.exe comp t6 src/t6/test.gsc
[ERROR]:compiler:src/t6/test.gsc:5:1: syntax error, unexpected } at src/t6/test.gsc
```

Example working unexpectedly:
```cpp
// file path: "src/t6/test.gsc"

init()
{
    println("foo1")
    println("foo2");
}
```
```bash
./bin/gsc-tool.exe comp t6 src/t6/test.gsc
compiled t6/test.gsc
```

Game error:
![game-error](https://user-images.githubusercontent.com/43913951/229453626-e4511f0e-d201-45f4-b570-ba9f185685b1.png)

Console error:
![console-error](https://user-images.githubusercontent.com/43913951/229453643-6f4a4091-0e19-4980-8486-1e9dae51be93.PNG)

**Comment by xensik** [comment]
im already aware of this. the syntax in second example is correct. the call gets treated as a object call ''obj func(args);" where obj is another function call. so ``println("foo1") println("foo2");`` is a method call. im thinking about enforcing the object to be in the same line of function to prevent this.

**Comment by xensik** [comment]
fixed on #123. error throws now if the object is on a different line than function, example:
```c
obj
func();
```

**Comment by arnauruana** [comment]
It works fine now. Thank you very much.
